### PR TITLE
Adds metrics middleware v4

### PR DESCRIPTION
**Pull request by joesantos418**
Adds a new version of the metrics middleware with predefined metrics prefixed with "fkit_" to avoid problems with a collector overwriting labels.

### CODE REVIEW

File: gokitmiddlewares/metricsmiddleware/v4/config.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	return c
+}
+
+// NewDefaultConfig returns a new Config with sane defaults.
+func NewDefaultConfig(endpoint string) Config {
+	config := Config{
+		Metrifier: metrifier.NewDefaultConfig("endpoint", ""),
+	}
+	config.Metrifier.ConstLabels = map[string]string{
+		"fikt_endpoint": endpoint,
```
Comment (eric-reis) [inline gokitmiddlewares/metricsmiddleware/v4/config.go:41]: RC: `fkit` is written wrong here

**Comment by eric-reis** [comment]
Reviewing finished. There's a typo that has impact in the solution and thus must be corrected before merge

File: gokitmiddlewares/metricsmiddleware/v4/config.go (hunk 1 above)
Comment (joesantos418) [reply gokitmiddlewares/metricsmiddleware/v4/config.go]: Fixed it, thank you for noticing

**Review by eric-reis — APPROVED** [review]
Approving. `govulcheck` failure are caused by external dependencies, but they are better updated in other changesets.

**Pull request merged**